packages/standard-library.md

```diff
@@ -5,3 +5,175 @@ The Pony standard library is a collection of packages that can each be used as n
+    
+    for (key, value) in inhabitants_count_per_country.pairs() do
+      env.out.print(key + " has " + value.string() + " inhabitants")
+    end
+```
+
+### The format package
+
+The format package is used to control how numbers are displayed
+
+```pony
+use "format"
+
+actor Main
+  new create(env: Env) =>
+    let the_answer = I32(42)
+    env.out.print(the_answer.string()) // 42
+    env.out.print(Format.int[I32](the_answer, FormatDefault )) //42
+    env.out.print(Format.int[I32](the_answer, FormatHex)) //0x2A
+    env.out.print(Format.int[I32](the_answer, FormatOctal)) //0o52
+    env.out.print(Format.int[I32](the_answer, FormatBinary)) //0b101010
+```
+
+### The regex package
+
+Obviously, the regex package deals with regular expressions.
+
+```pony
+use "regex"
+
+actor Main
+  new create(env: Env) =>
+    try
+      let r = Regex("(pony is) .*")?
+      let result = r.replace("pony is a kind of horse", "$1 awesome")?
+      env.out.print(result.clone())
+    end
+```
+
+### The file package
+
+The file package is pretty much what you expect. It deals with files and directories.
+
+```pony
+use "files"
+
+actor Main
+  new create(env: Env) =>
+    try
+      for file_name in env.args.slice(1).values() do
+        let path = FilePath(env.root as AmbientAuth, file_name)?
+        match OpenFile(path)
+        | let file: File =>
+          while file.errno() is FileOK do
+            env.out.write(file.read(1024))
+          end
+        else
+          env.err.print("Error opening file '" + file_name + "'")
+        end
+      end
+    end
+```
+
+### The time package
+
+The time package deals with date and time. This is interesting because scheduling is not preemptive in Pony.
+So if you want an actor to do a bit of work regularly, timers provide a good foundation for that.
+
+```pony
+use "time"
+
+// Count from 0 to 10 with a 1 second pause interleaved.
+
+actor Main
+  new create(env: Env) =>
+    let timers = Timers
+    let timer = Timer(Notify(env), 0, 1_000_000_000)
+    timers(consume timer)
+
+class Notify is TimerNotify
+  let _env: Env
+  var _counter: U32 = 0
+
+  new iso create(env: Env) =>
+    _env = env
+
+  fun ref apply(timer: Timer, count: U64): Bool =>
+    _env.out.print(_counter.string())
+    _counter = _counter + 1
+    if (_counter > 10) then
+        false
+    else
+        true
+    end
+```
+
+### Other packages
+
+This is enough for an introduction but as you will explore Pony you may find other packages really helpful.
+The different net packages - [net](https://stdlib.ponylang.org/net--index) [net/http](https://stdlib.ponylang.org/net-http--index) and [net/ssl](https://stdlib.ponylang.org/net-ssl--index) -  are used for anything that is network related.
+The [promise](https://stdlib.ponylang.org/promises--index) package will help you collect result of work done by others actor. The [options](https://stdlib.ponylang.org/options--index) one will help you parse and manage arguments from the command line.
```

Comment:
the `options` package is actually already deprecated in favor of the `cli` package.